src/sentry/web/frontend/setup_wizard.py

```diff
@@ -90,22 +93,43 @@ def get(self, request: HttpRequest, wizard_hash) -> HttpResponse:
 
         # Fetching or creating a token
         token = None
-        tokens = [
-            x
-            for x in ApiToken.objects.filter(user=request.user).all()
-            if "project:releases" in x.get_scopes()
-        ]
-        if not tokens:
-            token = ApiToken.objects.create(
-                user=request.user,
-                scope_list=["project:releases"],
-                refresh_token=None,
-                expires_at=None,
+        serialized_token = None
+
+        can_use_org_tokens = len(orgs) == 1 and features.has(
+            "organizations:org-auth-tokens", orgs[0], actor=request.user
+        )
+
+        if can_use_org_tokens:
+            org = orgs[0]
+            token_str = generate_token(org.slug, generate_region_url())
+            token_hashed = hash_token(token_str)
+            token = OrgAuthToken.objects.create(
+                name="Sentry-Wizard-Generated-Token",
```

Comment:
hmm, IMHO we can drop the hyphens here, this is human readable and can contain spaces, WDYT? We could also mirror the naming schema from docs:

```suggestion
                name="Generated by Wizard",
```

or even add the date (not sure from the top of my head how to do that in python, but basically "Generated by Wizard on 2023-08-12" or something like this)

For docs we use the name: `Generated by Docs for {Project Name} on 2023-08-03`

Reply:
I like it - makes things more transparent! 
I'll change it to `Generated by Sentry Wizard on {date.today()}`. I can't specify the project though as we don't know which project will be selected yet. But just the data alone makes sense.